packages/android_alarm_manager_plus/test/android_alarm_manager_test.dart

```diff
@@ -3,7 +3,6 @@
 // found in the LICENSE file.
 
 import 'dart:ui';
-
```

Comment:
Could you run melos run format?

Reply:
I have problem with melos ! I always geting this error when running `melos run format` :
```
The syntax of the command is incorrect.
Failed to format Dart files: exit code 255.
```